index.html

```diff
@@ -86,7 +86,10 @@ <h2 class="text-3xl font-bold mb-4">aliquet praesent sodades</h2>
     <section class="blog bg-gray-200 py-16">
         <div class="container mx-auto">
             <h2 class="text-3xl font-bold mb-4">Our Sponsors!</h2>
-            <!-- Sponsor info goes here -->
+<!--              <p class="text-lg"><a href="https://www.wa.gov.au/organisation/department-of-primary-industries-and-regional-development">Department of primary industries and regional development</a></p> -->
```

Comment:
why

Reply:
It doesn't lower performance whatsoever, don't worry about it

Reply:
Its dashed out text for might be future sponsors

Reply:
Its a comment that adds no value, remove it